crates/red_knot_vendored/vendor/typeshed/stdlib/typing.pyi

```diff
@@ -199,6 +197,7 @@ class _SpecialForm(_Final):
         def __or__(self, other: Any) -> _SpecialForm: ...
         def __ror__(self, other: Any) -> _SpecialForm: ...
 
+Any: _SpecialForm
```

Comment:
I needed this to get the `try_from_module_and_symbol` logic to kick in and handle the `typing.Any` reference as a special case.  I'm assuming this isn't kosher, though, since it's a direct edit to a vendored file.

Reply:
Yeah, we'll have to instead add the logic for detecting `Any` into `infer_assignment_definition`, without the "annotated as `_SpecialForm`" condition that is checked in `infer_annotated_assignment_definition`.